crates/cheatnet/src/forking/cache.rs

```diff
@@ -89,7 +89,7 @@ impl ToString for ForkCacheContent {
 
 #[derive(Debug)]
 pub struct ForkCache {
-    fork_cache_content: ForkCacheContent,
+    pub fork_cache_content: ForkCacheContent,
     cache_file: Utf8PathBuf,
 }
```

Comment:
Maybe we should just expose `compiled_contract_class` as a "view" method on `ForkCache`? This way we don't have to have this bit weird nested `pub` setup.